content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>"As the pixel intensity distribution suggests, there are many low (between around 0 and 20) <s>to</s> and very high (between around 200 and 240) pixel values.  "</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Nice catch, thanks. Fixed it!